grants/milestone-delivery-guidelines.md

```diff
@@ -0,0 +1,27 @@
+# Milestone Deliverables Guidelines
+
+We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. Please submit your milestones by email to grants@web3.foundation.
+
+The submission should contain the following information: 
+ * **Name of the grant project**
+ * **Link to the open-source code/delivery** (including the correct open-source license)
```

Comment:
"(including the correct open-source license)" - let me suggest alternative wording:

"Please ensure the repo has the correct open-source license."

Reply:
Integrated it into the Milestone Deliverables section